### PR TITLE
➕ Storybook

### PR DESCRIPTION
**Pull request by DNaunin**
• Install Storybook
• Remove example stories folder
• Create components folder with example Button component
• Set stories folder in .storybook/main.js
• Load example component in pages/index.tsx
• Ignore /storybook-static
• Add .eslintignore

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
Wow! Very creative 🖌

**Review by RonjaMeinecke — APPROVED** [review]
LGTM🚀

**Review by cboenke — APPROVED** [review]
LGTM 🚀

**Review by mrana17 — APPROVED** [review]
lgtm 🚀

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
```
Comment (SilviaBarcellini) [inline .storybook/main.js]: Are you importing the button from stories or from components?

**Review by sweisz — APPROVED** [review]
LGTM 👍

File: .storybook/main.js (hunk 1 above)
Comment (DNaunin) [reply .storybook/main.js]: Thanks, I had already changed that, but somehow my second commit did not go through, updated it now! 🎉

**Pull request merged**